sources/packages/backend/apps/api/src/route-controllers/cas-invoice/tests/e2e/cas-invoice.aest.controller.getInvoices.e2e-spec.ts

```diff
@@ -0,0 +1,271 @@
+            "SBSD",
+            300,
+            {
+              effectiveAmount: 250,
+            },
+          ),
+          createFakeDisbursementValue(
+            DisbursementValueType.BCTotalGrant,
+            "BCSG",
+            500,
+            { effectiveAmount: 400 },
+          ),
+        ],
+      },
+      {
+        offeringIntensity: OfferingIntensity.partTime,
+        casSupplierInitialValues: {
+          supplierStatus: SupplierStatus.VerifiedManually,
+          supplierNumber: "222222",
+        },
+        casInvoiceInitialValues: {
+          invoiceStatusUpdatedOn: addDays(
+            5,
+            CAS_INVOICE_STATUS_LAST_UPDATED_ON_START_DATE,
+          ),
+        },
+      },
+    );
+    const disbursementReceipt = casInvoice.disbursementReceipt;
+    const casSupplier = casInvoice.casSupplier;
+    await saveFakeInvoiceFromDisbursementReceipt(
+      db,
+      {
+        casInvoiceBatch,
+        creator: systemUsersService.systemUser,
+        provincialDisbursementReceipt: disbursementReceipt,
+        casSupplier,
+      },
+      {
+        casInvoiceInitialValues: {
+          invoiceStatus: CASInvoiceStatus.ManualIntervention,
+          invoiceStatusUpdatedOn: addDays(
+            2,
+            CAS_INVOICE_STATUS_LAST_UPDATED_ON_START_DATE,
+          ),
+        },
+      },
+    );
+    const thirdInvoice = await saveFakeInvoiceFromDisbursementReceipt(
+      db,
+      {
+        casInvoiceBatch,
+        creator: systemUsersService.systemUser,
+        provincialDisbursementReceipt: disbursementReceipt,
+        casSupplier,
+      },
+      {
+        casInvoiceInitialValues: {
+          invoiceStatus: CASInvoiceStatus.ManualIntervention,
+          invoiceStatusUpdatedOn: addDays(
+            10,
+            CAS_INVOICE_STATUS_LAST_UPDATED_ON_START_DATE,
+          ),
+        },
+      },
+    );
+    const fourthInvoice = await saveFakeInvoiceFromDisbursementReceipt(
+      db,
+      {
+        casInvoiceBatch,
+        creator: systemUsersService.systemUser,
+        provincialDisbursementReceipt: disbursementReceipt,
+        casSupplier,
+      },
+      {
+        casInvoiceInitialValues: {
+          invoiceStatus: CASInvoiceStatus.ManualIntervention,
+          invoiceStatusUpdatedOn: addDays(
+            20,
+            CAS_INVOICE_STATUS_LAST_UPDATED_ON_START_DATE,
+          ),
+        },
+      },
+    );
+    const endpoint = `/aest/cas-invoice?page=0&pageLimit=2&sortField=invoiceStatusUpdatedOn&sortOrder=DESC&invoiceStatusSearch=${CASInvoiceStatus.ManualIntervention}`;
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+
+    // Act/Assert
+    const response = await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK);
+    const invoices = response.body.results as CASInvoiceAPIOutDTO[];
+    expect(invoices).toStrictEqual([
+      {
+        errors: null,
```

Comment:
can we have the error populated as it is an important o/p of manual intervention.